examples/mongodbatlas_advanced_cluster/auto-scaling-per-shard/README.md

```diff
@@ -0,0 +1,63 @@
+# MongoDB Atlas Provider -- Sharded Cluster with Independent Shard Auto-scaling
+
+This example creates a Sharded Cluster with 2 shards defining electable and analytics nodes. Compute auto-scaling is enabled for both `electable_specs` and `analytics_specs`, while also leveraging the [New Sharding Configuration](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/advanced-cluster-new-sharding-schema) by defining each shard with its individual `replication_specs`. This enables scaling of each shard to be independent. Please reference the [Use Auto-Scaling Per Shard](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/advanced-cluster-new-sharding-schema#use-auto-scaling-per-shard) section for more details.
+
+
+## Dependencies
+
+* Terraform MongoDB Atlas Provider v1.23.0
+* A MongoDB Atlas account 
+
+```
+Terraform >= 0.13
```

Comment:
Should this be updated to maybe lowest supported TF version? Just noticed we have this across all advanced_cluster examples

Reply:
Good catch, would align with other examples and simply not mention it. In terms of `required_version = ">= 1.0"` also thinking we could remove them from our examples so we have single point of reference for terraform version support. WDYT? I can create a ticket for this.

Reply:
Yes, I agree we should remove this altogether and have a single point of reference